linux/library/src/flutter_embedder.cc

```diff
@@ -99,8 +101,8 @@ static void GLFWOnFlutterPlatformMessage(const FlutterPlatformMessage *message,
     return;
   }
 
-  Json::FastWriter writer;
-  std::string output = writer.write(response);
+  Json::StreamWriterBuilder wbuilder;
+  std::string output = Json::writeString(wbuilder, response);
```

Comment:
According to the docs this is creating and destroying a new writer under the hood, much like the code above. If we have to heap-allocate reader/writer objects anyway, maybe we should just cache them? Unless the embedding API sends this on arbitrary threads (which I don't think is the case) it would be pretty easy.

Reply:
At the moment everything should be running in the same thread, so it should be fine.